src/manifest.rs

```diff
@@ -475,6 +476,26 @@ impl LocalManifest {
         Manifest::find_file(&Some(self.path.clone()))
     }
 
+    /// Set the version of the manifest
+    pub fn set_version(&mut self, dry_run: bool, version: &Version) -> Result<()> {
+        let table_path = &["package".to_string()];
+        let table = self.manifest.get_table(table_path)?.as_table_mut();
+        if let Some(table) = table {
+            let version_str: Value = format!("{:?}", version.to_string())
+                .parse()
+                .expect("valid toml");
+            if let Some(v) = table.entry("version").as_value_mut() {
+                *v = toml_edit::decorated(version_str, " ", "");
```

Comment:
@hdevalke you could probably also use something like this:
```suggestion
        if let Some(table) = table {
            let version_str = toml_edit::value(version.to_string());
            if let Some(v) = table.entry("version").as_value_mut() {
                *v = (*version_str.as_value().unwrap()).clone();
```

Reply:
Thanks for the suggestion, I used part of your solution, but changed to a `match` statement for readability.